mtp_noms_ops/apps/settings/views.py

```diff
@@ -152,3 +152,38 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['prisons'] = self.request.user_prisons
         return context
+
+
+class JobInformationView(FormView):
+    title = _('Help us improve this service')
+    template_name = 'settings/job-information.html'
+    success_url = reverse_lazy('security:dashboard')
+    form_class = JobInformationForm
+
+    def get_success_url(self):
+        if 'next' in self.request.GET:
```

Comment:
does this get populated and does it actually do something? also, if we allow a redirection to happen, we need to validate the target url. c.f. [eligibility view](https://github.com/ministryofjustice/money-to-prisoners-noms-ops/blob/master/mtp_noms_ops/apps/security/views/eligibility.py#L42)

Reply:
It does something now!